.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Build Test'
+
+on:
+  pull_request:
+    branches: ["main"]
+    types: [opened, synchronize]
+
+jobs:
+  build-tauri:
+    permissions:
+      contents: write
+    strategy:
+      fail-fast: false
```

Comment:
Why do we not fail-fast?

Reply:
Agreed with your other comment, should be a smoke test, can it give value to see all platforms that fail, but will agree is not that necesarry for the smoketest.